extensions/ecc/sw-macros/src/lib.rs

```diff
@@ -90,6 +90,8 @@ pub fn sw_declare(input: TokenStream) -> TokenStream {
         create_extern_func!(hint_decompress_extern_func);
         create_extern_func!(hint_non_qr_extern_func);
 
+        let setup_function = syn::Ident::new(&format!("setup_sw_{}", struct_name), span.into());
```

Comment:
I am ever so slightly concerned `setup_sw_*` isn't unique enough for a no_mangle. I will rename it `setup_sw_extern_func` similar to the other extern functions

Reply:
done in d109b3d0a75d7ce98c8eea1955d184d94eb594c5